frontend/src/components/form/InputNumber.vue

```diff
@@ -28,6 +28,9 @@ const props = withDefaults(defineProps<Props>(), {
 // State
 const { errorMessage, value } = useField<number>(toRef(props, 'name'));
 const fieldActive: Ref<boolean> = ref(false);
+
+// Emits
+const emit = defineEmits(['blur', 'focus']);
```

Comment:
Consider: renaming to `onBlur` `onFocus` to prevent confusion with default event types.